demos/platforms/lpc4074.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright 2024 Khalil Estell
```

Comment:
Replace whole file with:

```C++
#include "lpc4078.cpp"
```

They are identical so we might as well copy and paste them using include.